src/components/DataInputForm.tsx

```diff
@@ -0,0 +1,74 @@
+
+import React, { useState } from 'react';
+import { View, TextInput, Button } from 'react-native';
+import { User } from '../types/UserTypes';
+import { DataInputApi } from '../apis/DataInputApi';
```

Comment:
There is no symbol `DataInputApi` in `../apis/DataInputApi` file. It rather exports `postDataInput ` method. Use that instead.